src/components/Form/Form.test.tsx

```diff
@@ -19,7 +21,7 @@ import {
 import { SubmitHandler } from "react-hook-form";
 import { ValidationFunction } from "../../types";
 import { DateInputValue, FrequencyValue } from "../../values";
-import { Frequency } from "..";
+import { Textarea } from "..";
```

Comment:
Although this  is an existing code , I am not feeling comfortable with `from "..";` Let's do `from "./Textarea";` instead

Reply:
Actually it should be imported the same way as the `Checkbox` and `Frequency` above. Which is from `../..`.

The reason they need to be imported from there is to ensure that the switch between deprecated and new form component works correctly